Firestore/README.md

```diff
@@ -16,6 +16,31 @@
   * In Xcode select the Firestore_IntegrationTests_iOS scheme
   * ⌘-u to build and run the integration tests
 
+### Running Integration Tests - using the Firestore Emulator
+
+Note: this does not give full coverage, but is much faster than above.
```

Comment:
Just curious, what makes it much faster?

Reply:
Several major factors:
  * Network: talking to `localhost` vs `firestore.googleapis.com`:
      * 50x lower latency: 0.033 ms RTT vs 17 ms
      * ~500x higher bandwidth: 47 Gbps vs 75 Mbps
      * No SSL to negotiate
  * Quorum: production implements a real database that does at least one paxos round
  * Durability: The emulator doesn't touch disk

This plays out in some very different numbers for common actions:
  * Write stream handshake: 0.169 ms vs 500 ms
  * Write stream single write: 0.143 ms vs 300 ms
  * Total test suite time: 15 sec vs 200 sec